components/molecules/tabs-filter.tsx

```diff
@@ -0,0 +1,54 @@
+import 'pure-react-carousel/dist/react-carousel.es.css'
+import {ButtonBack, ButtonNext, CarouselProvider, Slider} from 'pure-react-carousel'
+import {useRouter} from 'next/router'
+import Link from 'next/link'
+import classNames from '../../utils/classes'
+
+export type TabsFilterProps = {
+  tags: Array<string>
+}
+
+export default function TabsFilter({tags}: TabsFilterProps) {
+  const router = useRouter()
+  const tagsWithCategoryAll = ['All', ...tags]
```

Comment:
Don't need to create a new variable just replace your line by `tags.unshift('All')`